features/names.feature

```diff
@@ -85,3 +85,35 @@ Feature: BibTeX Names
     | {Barnes} {and} {Noble,} {Inc.}    | {Barnes} {and} {Noble,} |                | {Inc.}                         |     |
     | Charles Louis Xavier Joseph de la Vallee Poussin | Charles Louis Xavier Joseph | de la | Vallee Poussin       |     |
 
+	@names @display
+	Scenarios: Some Japanese names (ordinal names to weird cases)
+		| name                              | first                   | von            | last                           | jr  |
```

Comment:
In my editor these are correctly aligned. The reason of misalignment is that github uses `liberation mono` which doesn't include Asian characters, and our browser falls back to another font with different width (in my case Japanese font, and in you case perhaps a generic font that covers vast unicode chracters). 

Anyway, to avoid inconveniences, maybe another test file should be created for Asian characters.